common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java

```diff
@@ -19,5 +19,8 @@
 
     RenderEvent.ElementType renderType();
 
-    RenderState renderAt() default RenderState.POST;
+    RenderState renderAt() default RenderState.PRE;
```

Comment:
I surely don't want this to be changed.. We've set every overlay with the assumption this is Post, and specified PRE for specific cases. I've seen the chat problem, and most likely a better fix would be is to just patch the chat background to not be "solid".

Reply:
Even if it seems working, we would have to test tens of overlays manually, and make sure it does not interact weirdly with anything, which is problematic, and WILL lead to bugs.

Reply:
Not to mention, this does not fix any overlays' rendering issues that need to be rendered on Post for some reason.